### PR TITLE
Fix issue with ImageNormalize not working with Matplotlib < 1.2 because Normalize used to be an old-style class

### PR DESCRIPTION
**Pull request by astrofrog**
Will fix https://github.com/astropy/photutils/issues/247

### CODE REVIEW

**Comment by astrofrog** [comment]
@mdboom - as the resident matplotlib expert, does this seem reasonable? :)

**Comment by embray** [comment]
You might want to wait to merge #3389 and then rebase this on that :)

File: astropy/visualization/mpl_normalize.py [hunk 1]
```diff
@@ -4,12 +4,18 @@
...
 from __future__ import division, print_function
 
+from distutils.version import LooseVersion
+
 import numpy as np
 from numpy import ma
 
 try:
+    import matplotlib
+    MATPLOTLIB_INSTALLED = True
```
Comment (embray) [inline astropy/visualization/mpl_normalize.py]: Not that it matters, but maybe use `HAS_MATPLOTLIB`--more consistent with the style used elsewhere.

**Comment by mdboom** [comment]
1.2 is really old, but if you need it, I think this is a reasonable workaround.

**Comment by astrofrog** [comment]
@mdboom - this was reported by a user so I think it's not entirely unlikely that older versions are installed in some cases, and it's an easy fix.

@embray - I'd like to merge this before #3389 because this needs to be backported for the next RC but I'm not sure if we are backporting #3389? (if we are, then I'm fine with merging #3389 first then rebasing this).

**Comment by embray** [comment]
Either way--just thought I'd point it out.  Go ahead and merge this first.

**Comment by astrofrog** [comment]
Ok, thanks!

**Comment by embray** [comment]
Oh, now that I look at what this is doing there's a more straightforward way in the first place based on ducktyping.

**Pull request merged**